src/discljord/messaging.clj

```diff
@@ -596,13 +596,47 @@
   []
   [style])
 
+;; --------------------------------------------------
+;; Scheduled Events
+
+(defendpoint list-guild-scheduled-events! ::ds/guild-id
+  "Returns a promise containing a list of scheduled events for the given guild"
+  []
+  [with-user-count])
+
+(defendpoint create-guild-scheduled-event! ::ds/guild-id
+  "Creates a guild scheduled event, returns the resulting event object in a promise on success."
+  [ms.event/name privacy-level scheduled-start-time entity-type]
+  [channel-id entity-metadata scheduled-end-time ms.event/description ms.event/image])
+
+(defendpoint get-guild-scheduled-event! ::ds/guild-id
+  "Returns a promise containing the scheduled event with the given id for the given guild."
+  [event-id]
+  [with-user-count])
+
+(defendpoint modify-guild-scheduled-event! ::ds/guild-id
+  "Modifies the given parameters of the scheduled event with the given id for the given guild."
+  [event-id]
+  [channel-id entity-metadata ms.event/name privacy-level scheduled-start-time scheduled-end-time
+   ms.event/description entity-type ms.event/status ms.event/image])
+
+(defendpoint delete-guild-scheduled-event! ::ds/guild-id
+  "Deletes scheduled event with the given id, returning a promise of whether it succeeded."
+  [event-id]
+  [])
+
+(defendpoint get-guild-scheduled-event-users! ::ds/guild-id
+  "Returns a promise containing a list of event users subscribed to the given event."
+  [event-id]
+  [limit with-member before after])
+
 ;; --------------------------------------------------
 ;; Invite
 
 (defendpoint get-invite! nil
   "Returns a promise containing the invite."
   [invite-code]
-  [with-counts?])
+  [with-counts])
```

Comment:
I think this is a breaking change. What was the intended result of this change here?

Reply:
This is actually a fix - `with-counts?` was passed as literally "with_counts?" in the query parameters, which did nothing because it's called "with_counts". 
So code that previously used `:with-counts?` instead of `:with-counts` didn't do anything anyway.

Reply:
Ah, I think I made a mistake in the original dispatch for this then, I had thought I corrected that parameter. This change is good then.